### ISSUE TITLE
Same key for different references in private and group library

### ISSUE THREAD

**Issue by tbsmn** [retorquere/zotero-better-bibtex#1015]
- [x] I have verified in the Zotero addons screen that I have the latest release of BBT (https://github.com/retorquere/zotero-better-bibtex/releases/latest) and Zotero installed
- [x] I am posting a single question, bug or feature request
- [x] I have included a descriptive subject of the problem
- [x] I am available for follow-up questions and testing
- [x] If I am posting an export or key-generation related bug report, I have included an error-report by right-clicking the problematic reference and submitting an BBT error report

# Bug classification

## Exporting references
Report ID: DXBK5LAB

I have this Reference in my personal library: "Baumann, Kristian; Peter Klein; Antonija Mrsic Carl, and Denise Bender. 2014."
And I have this Reference in a group library: "Baumann, Silke. 2014." 

I use the automatic key generation with [auth][year] and have the postfixed option activated for conflicts. But both references get the same Key "Baumann2014". Shouldn't one of them be a key with a postfix like "Baumann2014a"

Thank you very much

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.0.155.7338](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.0.155.7338.xpi) ("permit selectable uniqueness scope").

**Comment by retorquere** [comment]
The behavior you see was by design; each library was seen as a separate key scope. 7338 adds a preference where you can select uniqueness scope between library and global. Please try if that does what you need (you'll need to refresh the key after installation to see the effect).

**Comment by tbsmn** [comment]
Thanks for the quick response. I just checked it out. However refreshing the key did not change it. I also realised that the Biblatex-Panel in Zotero-settings disappeard in this version.

**Comment by retorquere** [comment]
If it's disappeared, that means there's a startup error; the refresh won't do anything until you manage to change the preference. Can you see if you can submit a BBT debug log from the Help menu?

**Comment by tbsmn** [comment]
Debug log ID DW2LIQ64
I tried to switch back to the stable version and now the preference panel has disapeared here to, maybe that helps tracking down the problem.

**Comment by retorquere** [comment]
No error in the logs. Have you tried cmd-Q uitting Zotero and starting it again?

**Comment by tbsmn** [comment]
yes multiple times uninstalled and installed the plugin too. Gonna try a complete system reboot now.

**Comment by tbsmn** [comment]
OK even rebooting the system did not help :-(

**Comment by retorquere** [comment]
Can you enable extended debugging from the help menu, call up the prefs pane, and then submit a debug log?

**Comment by tbsmn** [comment]
Report ID = NTQTMRNT

**Comment by retorquere** [comment]
You're using the German translations, which I didn't add to this branch. Hold on.

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.0.155.7355](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.0.155.7355.xpi) ("New Crowdin translations (#1016)\n\n* New translations zotero-better-bibtex.dtd (French)\r\n\r\n* New translations zotero-better-bibtex.dtd (German)").

**Comment by tbsmn** [comment]
Thank you! That did the trick!

**Comment by retorquere** [comment]
I still need to merge this into a release.
